client/src/components/Atoms/CSVForm.tsx

```diff
@@ -12,12 +12,16 @@ import {
 import * as Yup from 'yup'
 import { CvrFileType, IFileInfo, FileProcessingStatus } from '../useCSV'
 import FormWrapper from './Form/FormWrapper'
-import FormSection, { FormSectionDescription } from './Form/FormSection'
+import { FormSectionDescription } from './Form/FormSection'
 import { ErrorLabel, SuccessLabel } from './Form/_helpers'
 import { sum } from '../../utils/number'
 import FormButton from './Form/FormButton'
 import AsyncButton from './AsyncButton'
 
+// CSVFile is deprecated in favor of FileUpload
+
+const FormSection = styled.div``
```

Comment:
Can probably just drop `FormSection` in favor of plain divs since we aren't applying any styles (unless I'm missing something or you intend to add styles soon)

Reply:
Here I was actually _removing_ the styles that are built into the existing `FormSection` component, but I didn't want to impact other forms that were relying on it. I'm hoping to replace the last instances of the `CSVForm` component soon, so all this code can be removed

Reply:
Switched to plain divs